app/services/xccdf_report_parser.rb

```diff
@@ -30,6 +29,10 @@ def check_report_format
     )
   end
 
+  def valid_message_format?(message)
```

Comment:
It might seem odd to have a separate method for this but I think it's likely we'll have to validate other fields in the message in the future - so just making this method a bit bigger would work for that